python-packages/smithy-core/smithy_core/documents.py

```diff
@@ -86,20 +86,19 @@ def __init__(
                 self._raw_value = value
             else:
                 self._value = value  # type: ignore
-        elif isinstance(value, Sequence):
+        elif isinstance(value, Sequence) and not isinstance(value, str | bytes):
```

Comment:
This is the first time I've seen this. Was this a change with typing that enabled usage of this syntax vs a tuple of types?

Reply:
Yeah, [PEP604](https://peps.python.org/pep-0604/), which added the union syntax, also made it useable in isinstance checks.